.github/workflows/appimage.yml

```diff
@@ -0,0 +1,53 @@
+name: AppImage
+
+on:
+  pull_request:
+    branches: [master, features]
+  release:
+
+jobs:
+  build-appimage:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Prepare environment
+        run: |
+          sudo apt update
+          sudo apt install python3 ruby
```

Comment:
These packages are already preinstalled in GitHub images: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md

```suggestion
```